tests-functional/clients/statusgo_container.py

```diff
@@ -331,6 +332,24 @@ def acquire_port():
         Config.status_backend_port_range.remove(host_port)
         return host_port
 
+    def connect_to_bridge_network(self):
+        if not self.container:
+            return
+        try:
+            networks_attached = set(self.container.attrs.get("NetworkSettings", {}).get("Networks", {}).keys() or [])
+            if "bridge" not in networks_attached:
+                try:
+                    bridge_net = self.docker_client.networks.get("bridge")
+                    bridge_net.connect(self.container)
+                    logging.info(f"Connected container {self.container.name} to bridge network")
+                except docker.errors.APIError as e:
+                    if "already exists" in str(e).lower():
+                        logging.debug(f"Bridge connection already exists for {self.container.name}")
+                    else:
+                        logging.warning(f"Failed to connect container to bridge network: {e}")
+        except Exception as e:
+            logging.warning(f"Bridge network attach check failed: {e}")
```

Comment:
Looks like this can be simplified

```suggestion
    def connect_to_bridge_network(self):
        if not self.container:
            return
            
        networks_attached = set(self.container.attrs.get("NetworkSettings", {}).get("Networks", {}).keys() or [])
        if "bridge in networks_attached:
           return

        try:
            bridge_net = self.docker_client.networks.get("bridge")
            bridge_net.connect(self.container)
            logging.info(f"Connected container {self.container.name} to bridge network")
        except docker.errors.APIError as e:
            if "already exists" in str(e).lower():
                # Not an error
                logging.debug(f"Bridge connection already exists for {self.container.name}")
                return

            # Otherwise re-raise the exception
            raise e
```

Reply:
Done, thanks